src/components/UserWarning/UserWarning.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export const UserWarning: React.FC = () => (
+  <section className="section">
+    <p className="box is-size-3">
+      Please get your
+      {' '}
```

Comment:
Instead of using {' '} for adding spaces, you can use a template literal for better readability.

Reply:
<img width="616" alt="image" src="https://github.com/mate-academy/react_todo-app-add-and-delete/assets/17066980/30a900aa-bdfb-4c4d-8247-e8b89108311f">